app/src/main/java/de/danoeh/antennapod/activity/OnlineFeedViewActivity.java

```diff
@@ -227,15 +227,14 @@ public void finish() {
 
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
-        switch (item.getItemId()) {
-            case android.R.id.home:
-                Intent destIntent = new Intent(this, MainActivity.class);
-                if (NavUtils.shouldUpRecreateTask(this, destIntent)) {
-                    startActivity(destIntent);
-                } else {
-                    NavUtils.navigateUpFromSameTask(this);
-                }
-                return true;
+        if (item.getItemId() == android.R.id.home) {
```

Comment:
This is unused. The activity does not have an options menu, so it can be removed completely.

Reply:
Sorry, I'd rather not make changes I wasn't intending to make as part of this pull request. If you'd like to do that, I'll leave it to you.